bagelbids/tests/test_cli_add_bids.py

```diff
@@ -0,0 +1,20 @@
+from bagelbids.cli import bagel
+
+
+def test_valid_inputs_run_successfully(
```

Comment:
optional toppings :cherries: (?):
put the `(add_)bids` command name in the test name? I know we can probably tell from the test module name what's being tested here, but I'm not sure if the test module name always shows up in the test summary.

Reply:
Pytest should always show the specific module where the failed test is located at the end of the test summary, e.g.
```
======================================================== short test summary info ========================================================= 
FAILED tests\test_cli_add_bids.py::test_bids_valid_inputs_run_successfully - AssertionError: Errored out. STDOUT: Usage: root [OPTIONS] COMMAND [ARGS]...
```
but that being said I think it never hurts to be extra clear in the naming considering how many tests we have and just to make our lives easier. 🙂